src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -239,7 +241,7 @@ private void GroupFixes(Workspace workspace, IEnumerable<CodeFixCollection> fixC
                             if (fix.Action is SuppressionCodeAction)
                             {
                                 var suggestedAction = new SuppressionSuggestedAction(workspace, _subjectBuffer, _owner._editHandler,
-                                    fix, fixCollection.Provider);
+                                    fix, fixCollection.Provider, getFixAllSuggestedActionSet);
```

Comment:
Because the set depends on indivudal nested action, see http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Implementation/Suggestions/SuppressionSuggestedAction.cs,69

We can loop over the actions and generate set here, but we already do so to compute the nested suggested actions, so I just do it in that loop.